cmd/run.go

```diff
@@ -234,7 +234,7 @@ func (r *Runner) Run(ctx context.Context, patterns []string) error {
 		if err != nil {
 			return fmt.Errorf("could not start proxy server: %w", err)
 		}
-		r.config.HTTPProxyURL = "https://" + proxyServer.Address
+		r.config.HTTPProxyURL = "http://" + proxyServer.Address
```

Comment:
Hrmm, IIRC for proxy URLs in the ways I've seen them used, usually `http` or `https` signify whether they are used to proxy HTTP or HTTPS connections, not whether they are actually behind TLS themselves. But I may be wrong.